modules/RTC/TPCUtils.js

```diff
@@ -723,10 +723,16 @@ export class TPCUtils {
 
         return videoCodec === CodecMimeType.VP8 // VP8 always
 
-            // K-SVC mode for VP9 when no scalability mode is set. Though only one outbound-rtp stream is present,
-            // three separate encodings have to be configured.
+            // For FF: scalabilityModeEnabled is not supported and we have to use simulcast.
+            // For other browsers we use K-SVC mode for VP9 when no scalability mode is set. Although
+            // only one outbound-rtp stream is present, three separate encodings have to be configured.
             || (!this.codecSettings[videoCodec].scalabilityModeEnabled && videoCodec === CodecMimeType.VP9)
 
+            // FF uses simulcast with AV1.
+            || (!this.codecSettings[videoCodec].scalabilityModeEnabled
```

Comment:
Why do we need this extra check here? Can't we just remove the check for H.264 on line 739?

Reply:
Oh. Are you saying remove the av1, vp9 and h264 checks and leave only:
Vp8 || (!this.codecSettings[videoCodec].scalabilityModeEnabled && this.codecSettings[videoCodec].useSimulcast)

Reply:
ok. I'm confused. what is your proposal for changing the code.